### DISCUSSION TITLE
Custom pen thickness

### DISCUSSION THREAD

**Discussion by MishaPhysics** [xournalpp/xournalpp#5695] [Q&A] [answered]
Hi. I am searching for any way (even edditing some Xournal files) just to get the pen thickness bigger than "Very thin", but smaller than "Thin". Please, help me to understand how to to that if it is possible. I have found similar discussion where recompilation was suggested but maybe where is some easier way to do it. Sorry if there is no other way for now. I am using Xournal on Windows.

**Comment by PillowTrauma** [comment, marked as answer]
You can emulate custom pen thickness by turning on pressure sensitivity, and setting the pressure multiplier to minimum. You can then set the thickness of the pen with the minimum pressure setting. 

To get a pen between very thin and thin, I set my minimum pressure to 0.7 and the multiplier to 0.5, and use the thin pen. For the way I write, it is the same as writing with pressure sensitivity off, but ymmv.

**Reply by MishaPhysics** [reply]
Thanks. It really works correctly.
